ga4gh/server/__init__.py

```diff
@@ -6,6 +10,6 @@
 CORS(app, allow_headers='Content-Type')
 
 # TODO: Replace hard-coded value with a flag or conifg setting.
-app.config['MAX_CONTENT_LENGTH'] = 2 * 1024 * 1024 # 2MB 
+app.config['MAX_CONTENT_LENGTH'] = 2 * 1024 * 1024  # 2MB
 
```

Comment:
This looks a pep8 fix @kerrydc. It should be covered by #63 though, so it will just get fixed in the a merge or rebase.